### PR TITLE
Added starts of language detection. 

### PR DESCRIPTION
**Pull request by AlexSwensen**
I added some basic language detection.

It should also be noted that in order for the minified build to update, `gulp build` needs to be run twice otherwise changes won't persist. For that reason i have included the updated build files as well.

### CODE REVIEW

File: compile/unminified/ng-img-crop.js [hunk 1]
```diff
@@ -3230,7 +3233,13 @@ crop.directive('imgCrop', ['$timeout', 'cropHost', 'cropPubSub', function ($time
             };
 
             if (scope.chargement == null) {
-                scope.chargement = 'Chargement';
+                var language = window.navigator.userLanguage || window.navigator.language;
+                if(language.contains('fr')) {
+                    scope.chargement = 'Chargement';
```
Comment (rickerd) [inline compile/unminified/ng-img-crop.js]: I see scope here and $scope in the other language

Comment (AlexSwensen) [reply compile/unminified/ng-img-crop.js]: I just updated this pull request with the fix. Force of habit working in controllers.

**Comment by AlexSwensen** [comment]
All the files should be updated now. Sorry about that. :-)

File: source/js/ng-img-crop.js [hunk 2]
```diff
@@ -161,7 +161,13 @@ crop.directive('imgCrop', ['$timeout', 'cropHost', 'cropPubSub', function ($time
             };
 
             if (scope.chargement == null) {
-                scope.chargement = 'Chargement';
+                var language = window.navigator.userLanguage || window.navigator.language;
```
Comment (rickerd) [inline source/js/ng-img-crop.js:165]: I updated the behaviour a little so its more solid.

**Comment by rickerd** [comment]
@AlexSwensen can you pull and run gulp build and again?
It would be even better if you would squash all the commits to 1 perfect commit.

> git rebase -i HEAD~4
> (4 is the number of commits you want to squash)

If you squashed all the commits you can force push it to your branch again

> git push -f

**Comment by AlexSwensen** [comment]
Beautiful. Thankyou for this!

**Comment by AlexSwensen** [comment]
Created your function to make life easy.

**Review by rickerd — APPROVED** [review]
Superb!

File: source/js/ng-img-crop.js [hunk 3]
```diff
@@ -160,17 +160,37 @@ crop.directive('imgCrop', ['$timeout', 'cropHost', 'cropPubSub', function ($time
...
-                var msg = 'Loading'; // Default to English
-                if(language.contains('fr')) { // French Msg.
-                    msg = 'Chargement';
+                
+                switch(language) {
+                    case 'nl':
+                        return 'Aan het laden';
+                    break;
+                        
+                    case 'fr':
```
Comment (AlexSwensen) [inline source/js/ng-img-crop.js:172]: Correct me if i'm wrong, but im not sure this function will always return full strings. For example, various returns for English can be both `en` and `en-US`?

I believe the same is for French and several other languages as well.

[Mozilla Docs on NavigatorLanguage.language](https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language)

**Comment by rickerd** [comment]
Yes this could be a thing. I updated the switch with other language codes.
Also i build the minified code and altered the build a little.
A new release is coming today.

**Comment by AlexSwensen** [comment]
Awesome, good to hear!

**Pull request merged**